Framework/DataHandling/src/SaveNXcanSAS.cpp

```diff
@@ -382,7 +382,7 @@ void addData1D(H5::Group &data, Mantid::API::MatrixWorkspace_sptr workspace) {
 
   //-----------------------------------------
   // Add Q with units  + uncertainty definition
-  const auto &qValue = workspace->x(0);
+  const auto &qValue = workspace->points(0);
```

Comment:
Indeed, does not make a difference. `points` rturn by value but you can bind it to a const reference.